python/flexflow/torch/model.py

```diff
@@ -2278,18 +2384,15 @@ def _trace_model(self):
         if self.is_hf_model:
             from transformers.utils.fx import symbolic_trace as \
                 hf_symbolic_trace
-            # NOTE: We pass in a fixed `input_names` based on what is needed
-            # for MT5ForConditionalGeneration's forward pass
-            input_names = ["input_ids", "attention_mask", "decoder_input_ids"]
```

Comment:
This commit gets rid of this hard-coding. For example, for `BertForSequenceClassification`, we may want `input_names = ["input_ids", "attention_mask", "token_type_ids"]`, motivating why we should change `input_names` to an argument.